broker/src/main/java/io/zeebe/broker/system/partitions/ZeebePartition.java

```diff
@@ -610,39 +601,17 @@ private StreamProcessor createStreamProcessor(final ZeebeDb zeebeDb) {
     return CompletableActorFuture.completed(null);
   }
 
-  private void stepByStepClosing(
-      final CompletableActorFuture<Void> closingFuture, final List<AsyncClosable> actorsToClose) {
-    if (actorsToClose.isEmpty()) {
-      closingFuture.complete(null);
-      return;
-    }
-
-    final AsyncClosable asyncClosable = actorsToClose.remove(0);
```

Comment:
So actually we remove the closables already before I thought this was part of the fix?